src/python/pants/backend/python/tasks/select_interpreter.py

```diff
@@ -56,10 +56,6 @@ def product_types(cls):
     return [PythonInterpreter]
 
   def execute(self):
-    # NB: Downstream product consumers may need the selected interpreter for use with
```

Comment:
I think this comment still applies - see line 64 on the RHS. Perhaps the comment is in the wrong location.